code/datums/wires/syndicatebomb.dm

```diff
@@ -16,11 +16,15 @@
 
 /datum/wires/syndicatebomb/on_pulse(wire)
 	var/obj/machinery/syndicatebomb/B = holder
+	if(usr)
+		B.add_hiddenprint(usr) // could be from a signaller
```

Comment:
Is there any point in doing if(usr) if you are using user anyway for log_message without checking?

Reply:
see recent commits